src/styles/custom.scss

```diff
@@ -8,27 +6,5 @@
 @use "./fonts" as *;
 @use "./colors" as *;
 
-// Global styles and resets
-body {
-  @extend %primary-font;
-  // font-size: $font-size-base;
-  color: $black;
-  background-color: $white;
-  margin: 0;
-  padding: 0;
-}
```

Comment:
This directive was deleted but not moved to a different file like the other two -- should it have been?

Reply:
it was repeated in the base file, that was it was just deleted in custom.